production_configs/20240828_v0.10.9_GC_dec_min_2924_tuned/README.md

```diff
@@ -0,0 +1,11 @@
+# Prod 20240828_v0.10.9_GC_dec_min_2924_tuned
+
+## Description of the config
+
```

Comment:
If you could also provide the command you used to produce the config - it is very useful to other users / to reproduce / to debug, as well as a short description of why this production was requested.
Thanks.

Reply:
I have added commands for producing lstchain config and lstmcpipe config along with a short desciption of why this production is being requested.

Reply:
Thanks !